src/pages/Quiz/index.js

```diff
@@ -23,12 +31,27 @@ function Quiz() {
 
   return (
     <div className={classes.root + ' quizGame'}>
-      <Typography variant="h3">Quiz List</Typography>
-      {quizData.map((quiz, index) => (
-        <Button variant="outlined" size="large" href={`../quiz/${index}`}>
-          {quiz.title}
-        </Button>
-      ))}
+      <Paper elevation={2} className={classes.paper}>
+        <Typography variant="h5" gutterBottom>
+          Quiz List
+          <Divider />
+        </Typography>
+        <Grid container spacing={1}>
+          {quizData.map((quiz, index) => (
+            <Grid key={index} item xs={12}>
+              <Button
```

Comment:
Use `<Link />` component from `react-router-dom` library instead of Material UI `<Button />`.
It allows you to move to another page without refreshing React content.

Reply:
Fixed and will send update in the next commit

Reply:
Don't you think that you can use Link with a custom component instead of Button + Link?

According to your current implementation, I can see that you have the next structure (HTML structure):

```html
<button>
  <a href="/">Link</a>
</button>
```

But, what do you think about this structure?

```html
<button onClick="redirect">Link</button>
```

In this case, your HTML page will have a better semantic structure.

Take a look at [Docs](https://reactrouter.com/web/api/Link)

![image](https://user-images.githubusercontent.com/2904139/92712448-5a90b900-f362-11ea-9c21-c3e2385c1c4f.png)

You can try to provide a component (according to your code, it's a Material UI button) into Link.